src/applications/edu-benefits/feedback-tool/helpers.js

```diff
@@ -26,7 +26,7 @@ export function fetchInstitutions({ institutionQuery, page, onDone, onError }) {
 }
 
 export function transform(formConfig, form) {
-  const formData = transformForSubmit(formConfig, form);
+  const formData = transformForSubmit(formConfig, form, null);
```

Comment:
This will make the `transformForSubmit()` method no longer use the very aggression JSON stringify transformer that's removing international addresses that lack a zip/postal code (which included all int'l addresses added via the school search tool).

Reply:
the transformForSubmit method is defined [here](https://github.com/usds/us-forms-system/blob/master/lib/js/helpers.js#L382)
the stringify transformer I'm avoiding using is defined [here](https://github.com/usds/us-forms-system/blob/master/lib/js/helpers.js#L336)

Reply:
Would it make sense to create an issue on the us-forms-system repo to break the cleaning logic out of the stringify transformer?

Reply:
yeah, I talked to Anne about this late yesterday and she agrees that the problematic JSON stringify transformer code is ooooold and the transform logic should be refactored. We're not sure the best approach, but we should document this.